bin/generate-tiles

```diff
@@ -84,7 +86,13 @@ function run_tilelive_copy() {
 }
 
 
-if [[ -z "${MID_ZOOM-}" ]]; then
+if [[ -n "${LIST_FILE-}" ]]; then
+
+  # Generate all tiles given in a file LIST_FILE.
+  echo "$(date '+%Y-%m-%d %H-%M-%S') Generating tiles from a list $LIST_FILE from $HOST_COUNT servers, using $MAX_HOST_CONNECTIONS connections per server, $ALL_STREAMS streams"
+  run_tilelive_copy --scheme=list "--list=$LIST_FILE" --timeout="${TIMEOUT:-1800000}"
```

Comment:
```suggestion
  run_tilelive_copy --scheme=list --list="$LIST_FILE" --timeout="${TIMEOUT:-1800000}"
```

Reply:
In addition, 

1. it may be simpler to place `--timeout` within the `run_tilelive_copy` function rather than adding `--timeout="${TIMEOUT:-1800000}"` whenever it is called.
2. It may be more appropriate for the default `TIMEOUT` to be handled together with the other defaults at the beginning of the script

Reply:
the timeout default is different in the mid-zoom handling section, so it won't be as straightforward to send two different defaults into mid-zoom